x-pack/plugins/observability/typings/chart/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface ChartData {
+  time: number;
+  value: number;
+  group: string;
+}
+
+export interface SearchParams {
+  start: number;
+  end: number;
+}
```

Comment:
Do we want to enhance the `start` and `end` names to indicate that these numbers are supposed to be timestamps?

We probably also want to specify a unified bucket size so the charts line up properly, right? The individual data providers can still choose to ignore that parameter if it doesn't make sense it their domain.

```suggestion
export interface SearchParams {
  // The start timestamp in milliseconds of the queried time interval
  startTime: number;
  // The end timestamp in milliseconds of the queried time interval
  endTime: number;
  // The aggregation bucket size in milliseconds if applicable to the data source
  bucketSize: number;
}
```

Reply:
That makes sense, but I'll also change it in a next PR because there's a search box available on the page.
<img width="1350" alt="Screenshot 2020-06-10 at 13 30 42" src="https://user-images.githubusercontent.com/55978943/84262816-9a0af780-ab1e-11ea-8c98-60458b5342bb.png">